### PR TITLE
Add perftest Application Gateway CNAME record

### PR DESCRIPTION
**Pull request by paulridout**
### Jira link (if applicable)
N/A

### Change description ###
Add CNAME record for testing connection to perftest Application Gateway

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [no] Does this PR introduce a breaking change

### CODE REVIEW

File: environments/test/perftest.yml [hunk 1]
```diff
@@ -264,5 +264,6 @@ cname:
   - name: "cui-ra"
     ttl: 300
     record: "hmcts-perftest-c5g8ard0d4c5cdd9.z01.azurefd.net"
-
-
+  - name: "perftest.hmcts.net"
```
Comment (thomast1906) [inline environments/test/perftest.yml:267]:
```suggestion
  - name: "sdttest"
```

Comment (paulridout) [reply environments/test/perftest.yml]: Changed name to "sdt".

**Review by thomast1906 — APPROVED** [review]

File: environments/test/perftest.yml [hunk 2]
```diff
@@ -264,5 +264,6 @@ cname:
   - name: "cui-ra"
     ttl: 300
     record: "hmcts-perftest-c5g8ard0d4c5cdd9.z01.azurefd.net"
-
-
+  - name: "sdt"
+    ttl: 300
+    record: "cft-mtls-api-mgmt-appgw"
```
Comment (thomast1906) [inline environments/test/perftest.yml:269]: add full record FQDN

Comment (paulridout) [reply environments/test/perftest.yml]: Changed record property to "cft-mtls-api-mgmt-appgw.perftest.platform.hmcts.net".

**Pull request merged**